tests/s3/test_object_workflow_operations.py

```diff
@@ -727,3 +765,184 @@ def test_delete_objects_and_mention_1000_objects_7657(self):
+    def test_45504(self):
+        """Test GET operation for object with key name consisting of 1024 bytes"""
+        self.log.info(
+            "STARTED: Test GET operation for object with key name consisting "
+            "of 1024 bytes")
+        obj_key_name = "".join(
+            random.choices(
+                f"{string.ascii_letters}{string.digits}",
+                k=S3_OBJ_TST["test_8547"]["obj_key_length"]))
+        self.create_bucket_put_get_object(
+            self.bucket_name, obj_key_name, self.file_path,
+            S3_OBJ_TST["s3_object"]["mb_count"])
+        self.buckets_list.append(self.bucket_name)
+        self.log.info(
+            "ENDED: Test GET operation for object with key name consisting "
+            "of 1024 bytes")
+
+    @pytest.mark.parallel
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_ops
+    @pytest.mark.tags("TEST-45506")
+    def test_45506(self):
+        """Test GET operation for object with key name consisting of prefix and delimiter"""
+        self.log.info(
+            "STARTED: Test GET operation for object with key name consisting "
+            "of prefix and delimiter")
+        obj_key_name = "Finance/statements"
+        self.create_bucket_put_get_object(
+            self.bucket_name, obj_key_name, self.file_path,
+            S3_OBJ_TST["s3_object"]["mb_count"])
+        self.buckets_list.append(self.bucket_name)
+        self.log.info(
+            "ENDED: Test GET operation for object with key name consisting "
+            "of prefix and delimiter")
+
+    @pytest.mark.parallel
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_ops
+    @pytest.mark.tags("TEST-45507")
+    def test_45507(self):
+        """Test GET operation for object with key name consisting of alphanumeric chars"""
+        self.log.info(
+            "STARTED: Test GET operation for object with key name consisting "
+            "of alphanumeric chars")
+        count_limit = self.system_random.randrange(4,10)
+        letters = "".join(random.choices(string.ascii_letters, k=count_limit))
+        digits = "".join(random.choices(string.digits, k=count_limit))
+        obj_key_name = f"{letters}{digits}"
+        self.create_bucket_put_get_object(
+            self.bucket_name, obj_key_name, self.file_path,
+            S3_OBJ_TST["s3_object"]["mb_count"])
+        self.buckets_list.append(self.bucket_name)
+        self.log.info(
+            "ENDED: Test GET operation for object with key name consisting "
+            "of alphanumeric chars")
+
+    @pytest.mark.parallel
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_ops
+    @pytest.mark.tags("TEST-45508")
+    def test_45508(self):
+        """Test GET operation for object with key name consisting of underscore(_)"""
+        self.log.info(
+            "STARTED: Test GET operation for object with key name consisting "
+            "of underscore(_)")
+        obj_key_name = ["_objworkflow", "objworkflow_", "_objworkflow_", "obj_workflow"]
```

Comment:
instead of fixed string, use random string generator and then append/prepend/insert the special character `-`

Reply:
done